app/jobs/import_talroo_jobs_job.rb

```diff
@@ -0,0 +1,7 @@
+class ImportTalrooJobsJob < ApplicationJob
+  queue_as :default
```

Comment:
I wonder if we should set up a different queue for these, or at least set it to low

Reply:
I think we setup a new queue with lower priority than `low`